hmy/api_backend.go

```diff
@@ -27,7 +28,12 @@ import (
 
 // APIBackend An implementation of internal/hmyapi/Backend. Full client.
 type APIBackend struct {
-	hmy *Harmony
+	hmy              *Harmony
+	MedianStakeCache struct {
+		Mut            sync.Mutex
```

Comment:
No need it give it a name at all, just embed the Mutex alone, sync.Mutex, then we can do MedianStakeCache.Lock() directly.